tensorrt_llm/_torch/visual_gen/ubx_alltoall.py

```
@@ -0,0 +1,175 @@
+        if self._init_err is not None:
+            return False
+        if self._allocator is not None:
+            return True
+        try:
+            from ubx.allocator import SymmAllocator
+
+            pool_bytes = self._POOL_MB * 1024 * 1024
+            self._allocator = SymmAllocator(pool_bytes, device, self._pg)
+            logger.info("UBXAllToAll: SymmAllocator ready")
+            return True
+        except Exception as exc:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)